.babelrc

```diff
@@ -1,4 +1,24 @@
 {
-  "plugins": ["transform-decorators-legacy"],
-  "presets": ["es2015", "stage-0", "react"]
+  "env": {
+    "development": {
+      "plugins": ["transform-decorators-legacy"],
+      "presets": ["es2015", "stage-0", "react"]
```

Comment:
Rather than use presets here let's just list plugins explicitly in every configuration? We can then move the shrared plugins up.